app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -1,5 +1,7 @@
 import React from "react";
-import { Checkbox } from "neetoui";
+import { Checkbox, Tooltip, Button, Badge, Avatar } from "neetoui";
+import dayjs from "dayjs";
+import tagOptions from "constants/tagOptions";
```

Comment:
Please rename the imported constant as `TAG_OPTIONS`.
Please refer [naming constants](https://github.com/airbnb/javascript#naming--uppercase) for more details.

Reply:
Also, please don't create aliases for constants. Rather, please create a `constants` file per feature. Like the `constants` file for notes feature can be inside the directory `app/javascript/src/components/Dashboard/Notes`.


However, you can create an alias for `constants` when you consider global constants that will be used throughout the application or at multiple locations.